python/lsst/ts/nightreport/database.py

```diff
@@ -0,0 +1,49 @@
+"""Configuration definition."""
+
+__all__ = ["NightReportDatabase"]
+
+import asyncio
+
+import sqlalchemy as sa
+import sqlalchemy.engine
+import structlog
+from sqlalchemy.ext.asyncio import create_async_engine
+
+
+class NightReportDatabase:
+    """Connection to the night report database and tables creation.
+
+    Creates report table if it doesn't exist.
```

Comment:
Please always use an active voice when writing doc strings. In this case `Creates` -> `Create`.